spec/factories/ads.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :ad do
+    address { "Somewhere far away" }
+    city { build(:city) }
+    consent { true }
+    description { "Some description." }
+    kind { :supply }
+    phone { "09876543210" }
+  end
+end
```

Comment:
Heh, in my WIP MR (I hope to open it today) I also added the factory so I'll get conflicts.

FYI, this is my definition:
```
FactoryBot.define do
  factory :ad do
    description { "Lore ipsum" }
    sequence(:phone) { |n| "098/555-#{n}" }
    sequence(:kind) { |n| Ad::KINDS[n % Ad::KINDS.size] }
    consent { true }
    address { "Ulica 1" }
    sequence(:email) { |n| "tester#{n}@example.com" }
    sequence(:category) { |n| Ad::CATEGORIES[n % Ad::CATEGORIES.size] }
    city
  end

  factory :city do
    sequence(:name) { |n| "City #{n}" }
    county
  end

  factory :county do
    sequence(:name) { |n| "County #{n}" }
  end
end
```
I put it all in one file since I don't expect this project will ever have many models, but I'm perfectly ok with the organisation you have here.

Reply:
No strong opinions so do as you prefer when resolving conflicts.